addons/controllers/testdata/test-cpi.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Can we rename this file to something like `test-cpi-nonparavirtual.yaml` that corresponds to `test-cpi-paravirtual.yaml` above?

Reply:
changed